### DISCUSSION TITLE
Install fail 4.14 rpm-ostree

### DISCUSSION THREAD

**Discussion by chris93111** [okd-project/okd#1831] [General]
Hello i try initial installation of okd 4.14 and controlplane not boot with error on rpm-ostree

He take the master ignition of machine api

crio is not present or not installed

bootstrap work correctly

Vsphere UPI / terraform

**Version**
openshift-install 4.14.0-0.okd-2023-12-01-225814
built from commit 82051a1c3d42e394be353bbe2eacf348ccb75970
release image quay.io/openshift/okd@sha256:4510c50834c07e71521a68c021320b5e44978245c8031a858f74328e54e89e1c
release architecture amd64

**Log bundle**
Dec 17 00:32:03 master01-ocp pedantic_feistel[2327]: W1217 00:32:03.544773    2329 firstboot_complete_machineconfig.go:65] error: failed to update OS to quay.io/openshift/okd-content@sha256:b44cf95aa0a0a12b679ed438bb1773cc9ab6b93421c322dddb389ef639cbb0d2 : error running rpm-ostree rebase --experimental ostree-unverified-registry:quay.io/openshift/okd-content@sha256:b44cf95aa0a0a12b679ed438bb1773cc9ab6b93421c322dddb389ef639cbb0d2: error: Creating importer: skopeo proxy unexpectedly exited during request method Initialize: exit status: 1
Dec 17 00:32:03 master01-ocp pedantic_feistel[2327]: setpriv: failed to parse reuid: 'rpm-ostree'
Dec 17 00:32:03 master01-ocp pedantic_feistel[2327]:
Dec 17 00:32:03 master01-ocp pedantic_feistel[2327]: : exit status 1
Dec 17 00:32:03 master01-ocp pedantic_feistel[2327]: I1217 00:32:03.544799    2329 firstboot_complete_machineconfig.go:66] Sleeping 1 minute for retry

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Comment by chris93111** [comment]
Hi @vrutkovs thanks for your response

is caused by change the nsswitch , without change the rpm-ostree work correctly, but after os update , the resolution not work and i need to change and remove resolve [!UNAVAIL=return] 

My server dns is a bind in rhel9

```

# In order of likelihood of use to accelerate lookup.
passwd:     files altfiles sss systemd
shadow:     files
group:      files altfiles sss systemd
hosts:      files myhostname resolve [!UNAVAIL=return] dns
services:   files sss
netgroup:   files sss
automount:  files sss

aliases:    files
ethers:     files
gshadow:    files
networks:   files dns
protocols:  files
publickey:  files
rpc:        files
```
the bundle 
[log-bundle-20231218004334.tar.gz](https://github.com/okd-project/okd/files/13698170/log-bundle-20231218004334.tar.gz)

thanks
